doc/nrf/protocols/thread/configuring.rst

```diff
@@ -36,10 +36,21 @@ Thread requires the following Zephyr modules to properly operate in the |NCS|:
 Enable OpenThread in the |NCS|
 ==============================
 
-To use the Thread protocol in the |NCS|, set the following Kconfig options:
+You can use the Thread protocol in the |NCS| in two ways: by utilizing the Zephyr networking layer, or without it by passing Thread frames directly to the nRF 802.15.4 radio drier.
+
+* To use the Thread protocol with Zephyr networking layer, set the following configs:
+
+  * :kconfig:option:`CONFIG_NETWORKING` - This option enables the generic link layer and the IP networking support.
+  * :kconfig:option:`CONFIG_NET_L2_OPENTHREAD` - This option enables the OpenThread stack required for the correct operation of the Thread protocol and allows you to use it.
+
+* To use the Thread protocol, and use nRF 802.15.4 radio directly, set the following configs:
+
+  * :kconfig:option:`CONFIG_OPENTHREAD` - This option enables the OpenThread stack and use the nRF 802.15.4 radio directly.
```

Comment:
This config is also required for the first option - though probably automatically enabled with `CONFIG_NET_L2_OPENTHREAD` isn't it? If yes, then the description is not correct.

Reply:
I thought setting/clearing `CONFIG_NET_L2_OPENTHREAD` is enough.

Reply:
I would simplify it as much as possible, for nrf-radio, set `CONFIG_OPENTHREAD`, for L2, set `CONFIG_NET_L2_OPENTHREAD` just simple like this.

Reply:
I've explained it better.